src/libs/printer.js

```diff
@@ -311,7 +315,7 @@ export function embed(path, print, textToDoc, options) {
         return null;
     }
 
-    function wrapCode(code) {
+    function wrapCode(code, double) {
```

Comment:
Please add a docstring explaining what `double`` does for future authors :-). Also, give it a default value of of `false`.